backend/api/v1/auth_service.go

```diff
@@ -284,6 +295,53 @@ func (s *AuthService) UpdateUser(ctx context.Context, request *v1pb.UpdateUserRe
 		passwordHashStr := string(passwordHash)
 		patch.PasswordHash = &passwordHashStr
 	}
+	if request.MfaCode != nil {
```

Comment:
Add more comments on these three use cases and how it is used.

Reply:
Updated some comments in `auth_service.proto` and `auth_service.go`